app.js

```diff
@@ -13,24 +13,22 @@ dotenv.config();
 const app = express();
 const port = 8080;
```

Comment:
```suggestion
const PORT = 8080;
```
해당 부분처럼 상수를 할당하는 const는 기존에 정해놨던 컨벤션대로 UPPERCASE와 SCREAMING_SNAKE_CASE를 혼용해서 변경해야 될 것 같습니다!

Reply:
네 저도 추후 유지보수나 명확한 의미 부여를 위해 상수를 사용하는게 좋을거 같습니다.
port뿐만 아니라, GOOGLE_OAUTH 관련 URL들도 상수로 분리하면 코드의 의도가 더 잘 드러날 것 같아 constants/constants.js 파일에 따로 정의해두었습니다.

Reply:
넵! 제가 따로 확인하지 못한 부분도 같이 수정해주셔서 감사합니다! :)